package.json

```diff
@@ -110,17 +112,20 @@
     "@angular/ssr": "^17.3.10",
     "@babel/runtime": "7.25.9",
     "@kolkov/ngx-gallery": "^2.0.1",
-    "@ng-bootstrap/ng-bootstrap": "^11.0.0",
+    "@ng-bootstrap/ng-bootstrap": "^12.0.0",
     "@ng-dynamic-forms/core": "^16.0.0",
     "@ng-dynamic-forms/ui-ng-bootstrap": "^16.0.0",
     "@ngrx/effects": "^17.1.1",
     "@ngrx/router-store": "^17.1.1",
     "@ngrx/store": "^17.1.1",
     "@ngx-translate/core": "^14.0.0",
     "@nicky-lenaers/ngx-scroll-to": "^14.0.0",
+    "@popperjs/core": "^2.9.2",
+    "@types/grecaptcha": "^3.0.4",
+    "angular-idle-preload": "3.0.0",
```

Comment:
@Wout-atmire : Please don't add `@types/grecaptcha` back in the main dependencies.  This was moved to `devDependencies` in #3553, and you are reverting those changes.

You've also reverted the **removal** of `angular-idle-preload`, which is no longer used in DSpace (again this was removed in #3553).